components/ScriptLibrary/include/LHVM/LHVM.h

```diff
@@ -20,36 +20,40 @@
 
 #pragma once
 
-#include <string>
-#include <vector>
-
 #include <LHVM/VMInstruction.h>
 #include <LHVM/VMScript.h>
+#include <string>
+#include <vector>
```

Comment:
I'm a big fan of sorting includes by having this order
* super class or matching header for cpp file
* blank line
* cstdlib
* blank line
* stdlib
* blank line
" projet

It prevents issues with header leaking.